content/terraform/v1.12.x/data/language-nav-data.json

```diff
@@ -1001,11 +1001,7 @@
       },
       { "title": "Locking", "path": "state/locking" },
       { "title": "Workspaces", "path": "state/workspaces" },
-      { "title": "Remote State", "path": "state/remote" },
-      {
```

Comment:
Since the article is about sensitive values in state _and_ plan files, I figure we move it to the top level!